### ISSUE TITLE
Only one workspace in two monitors config

### ISSUE THREAD

**Issue by sherlocode** [gh0stzk/dotfiles#136]
Hello! Firstly, I would like to thank you for your incredible work! This automatic ricer saves a lot of time and has a stunning setup/look! Thank you so much!

This time when I installed it, the workspaces were not previously defined (5 or 6 were configured the other times I used it, even using "super + {_,shift + }{1-9,0}") nor were they dynamically increasing. So, researching, I found on reddit a way to add workspaces as needed, and directly on the current monitor you are using, without having to edit the settings file, and also, it is possible to define the name in a rofi window as soon as the command is pressed. I tested it and it worked perfectly here.
I don't know if the workspace issue occurred with me due to the 2 monitors I'm using, but I would like to suggest this configuration for your rice, if you find it convenient. The following settings would be added to the sxhkdrc file:

// Add a new desktop on the current monitor with name set by rofi
super + plus (I added a "ctrl" to avoid conflict)
     bspc monitor -a $(rofi -dmenu)

// Remove the current desktop
super + minus (I added a "ctrl" to avoid conflict)
     bspc desktop -r

The original post where I found the configuration is:
https://www.reddit.com/r/bspwm/comments/gfyo83/spawn_new_workspacesdesktops_dynamically/

Thanks!

**Comment by gh0stzk** [comment]
Hello, I guess something must have gone wrong with your setup. In reality, regardless of the number of monitors you have, you should have 6 desktops by default.

I'll look at the link you left me.

meanwhile you "solved your problem"?
Did you read the wiki about adding more monitors? https://github.com/gh0stzk/dotfiles/wiki/Two-or-more-monitors-setup

Open the Theme.sh file in the "emilia" directory, for example. What does the part where it launches the polybar look like? Does it look like this?

```bash
launch_bars() {

	for mon in $(polybar --list-monitors | cut -d":" -f1); do
		MONITOR=$mon polybar -q emi-bar -c ${rice_dir}/config.ini &
	done

}
```
If it doesn't look like that, it's my fault, I pushed the changes, but I forgot that part, I uploaded them a few hours later.

**Comment by sherlocode** [comment]
Hello! Thank you for your attention!
The "Theme.sh" file is as described, and even so it only brought one workspace this time on each monitor. In previous installations, the 6 spaces came normally. But this time, the polybar for the second monitor came automatically (and that makes me very satisfied), whereas the other times I had to configure manually and individually rice by rice (the attempts I made to generate the polybar for the second monitor globally , mixed the workspaces of monitor 1 with monitor 2). The only adjustment I had to make was just the workspaces, and for me it's solved!
I don't know if anyone else will have this same occurrence, but I thought it would be interesting to record and suggest this possibility if it occurs!
Anyway, thank you for your hard work on this amazing project, and for your attention to this issue!

Ps.: this time I didn't check the configuration part for 2 monitors or more because the second polybar appeared automatically as I reported previously. If something bothers me over time in this matter, I will try the configuration suggested in the bspwmc file. Thank you again.
